### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by tzippy84** [jensrossbach/esphome-ferraris-meter#42] [Feedback]
Wow, I am really thankful for your effort! Have been struggling a bit to get this working and it has been sitting in my Unterverteilung for a while now, all connected. Worked out of the box after adapting my PIN and key! Thank you so much, I really appreciate it! Did you publish it somewhere on a blog or something like that? I just stumbled upon it through google.

**Comment by jensrossbach** [comment]
Thank you very much for your appreciation! I don't have a blog that advertises the component but I've described the project briefly on my homepage (German only): https://jensrossbach.github.io/projects/#ferraris-meter
